test/integration/cdn/test_application.rb

```diff
@@ -9,8 +9,7 @@
 
     @hanami_env       = ENV['HANAMI_ENV']
     ENV['HANAMI_ENV'] = 'production'
-
-    assert system("HANAMI_ENV=production bundle exec hanami assets precompile")
+    assert system("bundle exec hanami assets precompile")
```

Comment:
@lucasallan Why did you removed `HANAMI_ENV=production`? This is the closest simulation of a deployment scenario.